docs/jwst/pathloss/description.rst

```diff
@@ -36,7 +36,7 @@ It creates 2 pairs of 1-d arrays, a wavelength array (calculated from the WCS ap
 the index of the plane in the wavelength direction) and a pathloss array
 calculated by interpolating each plane of the pathloss cube at the position of
 the source (which is taken from datamodel).  There are pairs of these arrays for
-both pointsource and uniformsource data types.
+both point source and uniform source data types.
```

Comment:
I think the whole doc needs a review and updates, now that nothing is done with the 1-D corrections. For example, up in line 9 it says this is the correction "to apply to spectra when the 1-d extraction is performed." That's no longer true. It's being applied here and now, at the 2-D stage. And we also need to remove or reword the couple of mentions regarding MIRI corrections not yet being implemented: it's more appropriate to say that correction algorithms for MIRI MRS and LRS modes have not yet been developed (not implemented could be construed as indicating the algorithms exist, but we just haven't gotten around to putting them in yet.)